src/briefcase/commands/update.py

```diff
@@ -68,6 +68,7 @@ def __call__(
         update_resources: bool = False,
         update_support: bool = False,
         test_mode: bool = False,
+        config_overrides: list[str] | None = None,
```

Comment:
needs to be passed to `finalize()`?

Reply:
Again, Yes, but not any more with the revised approach that I've implemented.